components/Navigation/index.js

```diff
@@ -7,15 +7,27 @@ const StyledFooter = styled.footer`
   right: 0px;
   left: 0px;
   z-index: 1;
-  background-color: orange;
+  background-color: var(--dijon);
   display: flex;
   justify-content: space-around;
-  border: 1px solid black;
-  padding: 10px;
+  border: 1px solid grey;
+  padding: 5px;
+  box-shadow: rgba(0, 0, 0, 0.2) 0px -3px 0px inset,
+    rgba(0, 0, 0, 0.3) 0px 7px 13px -3px, rgba(0, 0, 0, 0.4) 0px 2px 4px;
 `;
 
 const StyledLink = styled(Link)`
   text-decoration: none;
+  color: black;
+  font-size: 1.5rem;
+  font-family: "Gill Sans";
```

Comment:
'Gill Sans' is already applied to the body in styles.js. But let me ask one question: What if the user hasn't installed this font on his system?